internal/store/project_ip_access_lists.go

```diff
@@ -41,6 +42,9 @@ type ProjectIPAccessListDeleter interface {
 
 // CreateProjectIPAccessList encapsulate the logic to manage different cloud providers.
 func (s *Store) CreateProjectIPAccessList(entries []*atlas.ProjectIPAccessList) (*atlas.ProjectIPAccessLists, error) {
+	if len(entries) == 0 {
```

Comment:
drive by, if the list is empty you get an index out of bound error

Reply:
Thanks for this fix!! 💯